tests/src/test/java/alluxio/client/rest/JobMasterClientRestApiTest.java

```diff
@@ -120,13 +120,10 @@ public void cancel() throws Exception {
   }
 
   @Test
-  public void list() throws Exception {
+  public void listEmpty() throws Exception {
     List<Long> empty = Lists.newArrayList();
-    Map<String, String> params = Maps.newHashMap();
-    params.put("name", "");
-    params.put("status", "");
```

Comment:
In `master`, these query parameters were incorrectly resulting in the following endpoint: `http://192.168.50.235:58346/api/v1/master/job/list?namestatus`

After these changes, it resulted in the following endpoint: `http://192.168.50.235:56745/api/v1/master/job/list?name&status`.

The REST handler which handles this endpoint has "null" error-handling which was previously being used by this test, whereas providing the values result in empty strings being assigned.

Hence I removed the parameters to preserve the "null" behaviour of the test. A proper test should be written which actually makes use of these parameters in the future (I would do it except that I lack the bandwidth to include it in this PR).